packages/widgets/src/components/StatisticCard/StatisticCard.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import userEvent from '@testing-library/user-event';
+import { screen } from '../../../customTest';
+import StatisticCard from './StatisticCard';
+
+describe('StatisticCard', () => {
```

Comment:
Test cases: 
- Renders with value and title
- Renders with value, title, and unit
- Renders with value, title, and icon
- Renders with value, title, unit, and icon
- Renders with an integer value (no tooltip) => Check there's no tooltip
- Renders with a non-numeric value (no tooltip) => Check there's no tooltip
- Renders with a numeric value (tooltip) => Check that the tooltip is present